index.html

```diff
@@ -0,0 +1,58 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>lengushuru Portoflio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600;700&family=Lato:wght@300;400;700&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <header>
+    <nav class="nav-bar">
+      <div class="page--logo">
+        <a class="page--link " href="#">lengushuru</a>
+      </div>
+      <div class="nav--menu" id="nav-toggle">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <div class="nav-menu" id="nav-menu">
+        <div class="nav-close" id="nav-close">
+          <i class="bx bx-x"></i>
+        </div>
+        <div class="nav-links">
+          <a class="nav-item" href="#portfolio">Portoflio</a>
+          <a class="nav-item" href="#about-me">About</a>
+          <a class="nav-item" href="#container">Contact</a>
+        </div>
+      </div>
+      <div class="desk-top">
+        <a class="nav-ite" href="#portfolio">Portoflio</a>
+        <a class="nav-ite" href="#about-me">About</a>
+        <a class="nav-ite" href="#container">Contact</a>
+      </div>
+    </nav>
+  </header>
+  <section class="hero">
+    <h1 class="main_title">Hey There.<br>
+      I’m Charles</h1>
+    <h3 class="abt_hero">I’am a Software Developer</h3>
+    <p class="description_text">I can help you build a product , feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.</p>
+    <ul class="socia_icons">
+      <li><a href="#"><img src="images/github.png" alt=""></a></li>
+      <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
+      <li><a href="#"><img src="images/peace-icon.png" alt=""></a></li>
+      <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector(2).png" alt=""></a></li>
+    </ul>
```

Comment:
- Hey, you don't seem to have added any of these images to your project and there's no image directory in your project. please, double-check again to ensure this issue is fixed so that the social media icons are shown and visible in your project.
---


- Please, kindly add the missing background image as seen in the Figma design. please, check the Figma design to see the missing image.

![Screenshot from 2023-01-12 14-48-45](https://user-images.githubusercontent.com/59397159/212083679-5c597803-cfe0-47c8-b15d-25d104d1af92.png)

Reply:
### README MIT LINK

- Please, let's update the MIT link by adding the license as the link is currently broken now to make our readme more professional.